soc/arm/nordic_nrf/nrf52/Kconfig.soc

```
@@ -121,4 +121,15 @@ config NRF52_ANOMALY_198_WORKAROUND
 	  This anomaly applies to IC revisions "Engineering B" up to "3", the most
 	  recent one.
 
+config NRF52_ANOMALY_109_WORKAROUND
+	bool "Anomaly 109 workaround"
+	default y
+	depends on SOC_NRF52832
+	depends on NRFX_SPIS || NRFX_SPIM || NRFX_TWIM || NRFX_PWM
+	help
+	  Due to Anomaly 109 the first byte sent out by these peripherals is
+	  sometimes wrong. This occurs when the system enters IDLE and stops the
+	  64MHz clock at the same time as the peripheral that is using DMA is started.
+	  This anomaly applies to IC revisions up to "3", the most recent one.
```

"the most recent one" may get outdated and I don't expect anyone to correct this text then.
```suggestion
	  This anomaly applies to IC revisions up to "3".
```

---

The concern is valid but the same text is present on the anomaly above